src/ln/peer_handler.rs

```diff
@@ -121,6 +121,13 @@ impl<Descriptor: SocketDescriptor> PeerManager<Descriptor> {
 		}
 	}
 
+	/// Get the list of the public keys of the connected peers with noise
```

Comment:
nit: maybe refer to it as "initial handshake", cause "noise" seems like a protocol detail that you dont need to expose.